docassemble/MotionToStayGeneral/data/questions/motiontostaygeneral.yml

```diff
@@ -446,28 +485,33 @@ fields:
   - no label: argument_summary
     input type: area 
     rows: 6
+#---
+#code: |
+#  if is_first_document and is_indigent:
+#    final_form_to_file = [general_cover_sheet, combined_motion, affidavitofindigency_attachment]
+#  elif is_first_document and is_indigent:
+#    final_form_to_file = [general_cover_sheet, combined_motion, affidavitofindigency_attachment]
+#  else:
+#    final_form_to_file = [general_cover_sheet, combined_motion]
+#---
+#code: |
+#  form_to_file_no_cover = combined_motion
 ---
-code: |
-  if is_first_document and is_indigent:
-    final_form_to_file = [general_cover_sheet, combined_motion, affidavitofindigency_attachment]
-  else:
-    final_form_to_file = [general_cover_sheet, combined_motion ]
----
-code: |
-  form_to_file_no_cover = combined_motion
----
-code: |
-  form_to_sign_temp = []
-  #if has_evidence:
-  #  form_to_sign_temp = [motion_attachment["preview"], exhibits]
-  #else:
-  #  form_to_sign_temp = [motion_attachment["preview"]]
-  # Dont need to show them the exhibits again--it can be slow w/ big addendum
-  form_to_sign_temp = [motion_attachment["preview"]]
-  if no_notice_of_appeal_filed:
-    form_to_sign_temp.append(Notice_of_Appeal['preview'])
-
-  form_to_sign = pdf_concatenate(form_to_sign_temp, filename="Motion_to_Stay.pdf")
+#code: |
```

Comment:
Now that it's working, I'd remove this commented old code. If you ever want to know what was happening before you can always look at the git history.

Reply:
Will do!